vocabularies/Analytics.xml

```diff
@@ -176,6 +179,114 @@
         </Annotation>
       </Annotations>
 
+      <Function Name="AutoExpand" IsBound="true" EntitySetPath="InputSet">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.IsSignature" />
+        <Annotation Term="Core.Description" String="`$apply` transformation that expands an unnamed leveled hierarchy with custom aggregation of certain properties" />
+        <Parameter Name="InputSet" Type="Collection(Edm.EntityType)" Nullable="false">
+          <Annotation Term="Core.Description" String="Entity set to be processed" />
+        </Parameter>
+        <Parameter Name="Levels" Type="Collection(Analytics.AutoExpandLevel)" Nullable="false">
+          <Annotation Term="Core.Description" String="Collection of aggregation levels forming the visible part of a leveled hierarchy" />
+          <Annotation Term="Core.LongDescription">
+            <String>Each element in the collection defines the property names for one level.
+            A property must not referenced by more than one level.
+            The first element in the collection defines the property names of the coarsest level,
+            the following elements define the property names of consecutively finer-grained aggregation levels.
+            The function result comprises the leveled hierarchy with these levels in preorder.
+            All referenced properties must be groupable.</String>
+          </Annotation>
+        </Parameter>
+        <Parameter Name="LeafLevel" Type="Collection(Edm.String)" Nullable="false">
+          <Annotation Term="Core.Description" String="A possibly empty set of property names that constitute, together with the property names from `Levels`, the leaf level of the leveled hierarchy" />
```

Comment:
What does "property names from `Levels`" mean exactly? To me, Levels looks like an array of objects which contain an array of strings, right?